client/signup/main.jsx

```diff
@@ -309,6 +314,22 @@ class Signup extends React.Component {
 			this.recordExcludeStepEvent( siteTypeStepName, siteTypeValue );
 		}
 
+		if ( 'undefined' !== typeof excludePlanStep ) {
+			let planStepName = '';
+			if ( 'site-launch' === flowName ) {
```

Comment:
It worries me when I see flow names in generic code. It looks like this is being implemented in a very specific way rather than a reusable way.

Reply:
I've removed specific references and made it more generic. Thanks for the tip.